core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -520,7 +520,7 @@ private String transformOMAScore( String omaScore) {
         } else if( omaScore.equalsIgnoreCase("N") || omaScore.equalsIgnoreCase("neutral")) {
             return "N";
         } else if( omaScore.equalsIgnoreCase("[sent]")) {
-            return ExtendedMutationUtil.NOT_AVAILABLE; // TODO temp workaround for invalid sent values
+            return "NA"; // TODO temp workaround for invalid sent values
```

Comment:
seem to be used for filling the field `mutation_event.LINK_XVAR` with values like `getma.org/?cm=var&var=hg19,2,167263066,C,T&fts=all`....which are not even valid links (was this the old mutation assessor page)? Maybe @n1zea144 can comment as well?